ahnlich/db/Cargo.toml

```diff
@@ -40,6 +40,7 @@ serde_json.workspace = true
 
 
 [dev-dependencies]
+loom = "0.7.2"
```

Comment:
Probably reintroduced by mistake

Reply:
yeah, probably from trying to sync with other branches